src/main/java/org/stellar/sdk/KeyPair.java

```diff
@@ -227,7 +234,9 @@ public byte[] sign(byte[] data) {
 
   /**
    * Sign the provided data with the keypair's private key and returns {@link DecoratedSignature}.
-   * @param data
+   *
+   * @param data the data to sign, typically the tx hash
```

Comment:
```suggestion
   * @param data the data to sign
```
It's the envelope that gets signed, I think?

Reply:
well, the [hash of the serialized tx xdr and networkid](https://github.com/stellar/java-stellar-sdk/blob/protocol_19/src/main/java/org/stellar/sdk/Transaction.java#L66-L70),  but that was not changed here, I just added a little color there, and am fine removing this per suggestion.